spec/install/gemfile/path_spec.rb

```diff
@@ -13,6 +13,21 @@
     should_be_installed("foo 1.0")
   end
 
+  it "resolves from prior lockfile for source path gems" do
+    build_lib "rack-obama", "1.0", :path => lib_path("same_path")
+    build_lib "net-ssh", "1.0",    :path => lib_path("same_path")
+
+    install_gemfile <<-G
+      path "#{lib_path("same_path")}/" do
+        gem 'rack-obama'
+        gem 'net-ssh'
+      end
+    G
+
+    bundle :check, :env => { "DEBUG" => 1 }
+    expect(out).to match(/Resolved from lockfile/)
```

Comment:
I added a debug message for me to test against but thought might be useful going forward in case we need to debug a future problem where one of the "changed?" checks seems to do the wrong thing for people.